docs/docs/developers/tutorials/codealong/first_fees.md

```diff
@@ -0,0 +1,199 @@
+
+Start the sandbox (L1, L2, but not the PXE) via: `aztec start --sandbox.noPXE`
+
+### Specifying the network URL
+
+When using the `aztec-wallet` CLI tool, the Aztec network node to connect to can be specified:
+
+```bash
+aztec-wallet --node-url <string>
+```
+
+The string currently defaults to the the local sandbox: `http://host.docker.internal:8080`, or can be a bootnode url.
+This can alternatively be specified via an environment variable: `AZTEC_NODE_URL`.
+
+The equivalent in Aztec.js:
+
+```javascript
+???
+```
+
+## Create Account Contract in a PXE
+
+**TODO: as snippet**
+
+An account on Aztec is a smart contract that specifies a method of authentication and a method of payment, allowing it to be used by the protocol to perform a transaction.
+
+For convenience, Aztec has implemented an account contract that authenticates transactions using Schnorr signatures. The contract class for a Schnorr account is pre-registered on the network to bootstrap first use. Ordinarily for a contract to be deployed, its class would have to be register with the network first.
+
+When a PXE creates a Schnorr account, there are three key things that occur:
+ - generation of keys privately
+ - registeration of a Schnorr account locally (effectively an instance of the Schnorr account class)
+   - at this stage the address of the account is known, even though it is not yet deployed
+ - initialization and deployment of the account contract to the network
+
+**Create the keys and account in the PXE**
+
+Lets first create the account but only register the instance in the pxe. Its address will be calculated, and in a later step we can deploy it counterfactually to the network.
+
+```bash
+aztec-wallet create-account --register-only -a main
+```
+
+The equivalent in Aztec.js:
+
+```javascript
+
+```
+
+Your PXE now has keys for an account that can be deployed on Aztec network(s).
+
+## Paying for an account deployment transaction
+
+To make transactions on the network, your account contract will need to specify a payment method of the enshrined asset, "Fee Juice".
+
+TODO: types of payment methods, one-liner each
+- FPCs
+  - Sponsored
+  - Public / Private
+- bridge and claim
+- Fee juice
+
+### Sponsored FPC
+
+To bootstrap first use, a sponsored fee paying contract (the canonical sponsored FPC) can be used to deploy your first account.
+
+**TODO: as snippet**
+A fee paying contract (FPC) effectively implements fee abstraction. It is a contract that pays for transactions of other accounts, when its own custom criteria is met.
+In the case of the canonical sponsored FPC, the only criteria is an upper bound on how much it sponsors an account's transactions. This will be enough to at least deploy an account.
+
+The PXE can be queried for the canonical sponsored FPC address, and then specified as the payment method.
+
+Via the CLI:
+
+```bash
+FPC_ADDRESS=`aztec-wallet --get-canonical-sponsored-fpc-address`
```

Comment:
Is this the actual flag?

I see:
```bash
josh in ~ λ aztec-wallet --get-canonical-sponsored-fpc-address
error: unknown option '--get-canonical-sponsored-fpc-address'
```

Reply:
It as only merged 1-2 days ago, and there hasn't been a release.